src/index.html

```diff
@@ -4,18 +4,55 @@
     <title>Webpack Starter Kit</title>
   </head>
   <body>
-    <article class='full-panel'>
-      <h1>Webpack Starter Kit</h1>
-      <img src="./images/turing-logo.png" alt="turing logo" height="100px" width="100px">
-    </article>
-    <article class="login-panel">
-      <h1>Login</h1>
-      <!-- <section> -->
-        <input class="login-input" placeholder="Enter Username">
-        <input class="login-input" placeholder="Enter Password">
-        <button type="submit">Login</button>
-      <!-- /</section> -->
-    </article>
-    <!-- Do not include the index.js or main.bundle.js file here - it is done automatically by the webpack server -->
+    <main>
+      <form class="login-form-panel">
+        <h1>Login</h1>
```

Comment:
Nice. Don't forget to hit these with labels if you ahve a chance. I found that `aria-label` helped me pass the lighthouse audit rather than the standard `label`